### PR TITLE
waleed/overall

### PR DESCRIPTION
**Pull request by waleedlugod**
Added layers. Would probably need to fix z-index of about once merged. Not sure how to transition from landing to articles so left as is (for now). Scrolling heights adjusted. Added blur and fade.

### CODE REVIEW

**Comment by waleedlugod** [comment]
Will fix scrolling behavior bug later. Can't seem to recreate it 🥲but ill see what I can do

**Comment by theguidon** [comment]
- [ ] can you change all instances of `years-50` in code to `50-years` HAHA not sure why ito ung id!
---
image

- [ ] can you also add a "Scroll to enter" and the down arrow sa earlier landing! not in the design but suggest to add here
- [ ] also can you make all elements smaller (logo, title, and subtitle); play around with size and comment/pm different versions and which ones you like!

---

- [ ] fix scrolling behavior (arrows or from sidebar)
- i noticed na it doesn't work rin from the sidebar HAHA maybe for the js, the band-aid solution is:

```text
// say article 1 start y is 200 and article 2 is 400
if window.y >= 195
  show article 1
if window.y >= 395
  show article 2
```

- basically don't use the exact y but make it show up earlier? wdyt


other than that, GREAT JOB WALEED <3<3<3
